plugins/Makefile

```diff
@@ -714,6 +714,7 @@ PLUGIN_FILES += $(filter-out eightfold/src/plugin.cpp,$(wildcard eightfold/src/*
 # modules/types which are present in other plugins
 EIGHTFOLD_CUSTOM = Comparator comparator
 
+# custom C++17 flags
```

Comment:
This is really not necessary. We do this for a bunch of other modules already.
No need to "explain" that it is there.

Reply:
Yeah you are triggering a lot of builds after one another so other servers might not always be up for it.

After all the CI jobs are finished/failed you can restart the ones. Which I had already done for you in the mean time ;)